packages/lib-classifier/src/store/ClassificationStore.spec.js

```diff
@@ -1,8 +1,84 @@
 import ClassificationStore from './ClassificationStore'
+import Subject from './Subject'
+
+import { types } from 'mobx-state-tree'
+
+let rootStore
+
+const RootStub = types
+  .model('RootStore', {
+    classifications: ClassificationStore,
+    projects: types.frozen(),
+    subjects: types.frozen(),
+    workflows: types.frozen() 
+  })
+
+const subjectStub = {
+  already_seen: true,
+  favorite: true,
+  finished_workflow: true,
+  id: '3333',
+  locations: [
+    { 'image/jpeg': 'https://panoptes-uploads.zooniverse.org/335/0/44a48dd2-23b3-4bb5-9aa4-0e803ac4fe6d.jpeg' }
+  ],
+  metadata: {},
+  retired: false,
```

Comment:
Maybe set this to true, so that we know we're testing against this value and not the default.

Reply:
Good point, let me update this